main.tf

```diff
@@ -207,8 +305,8 @@ resource "aws_ecs_service" "service" {
   dynamic "service_registries" {
     for_each = var.service_registry_arn == "" ? [] : [1]
     content {
-      registry_arn   = var.service_registry_arn
-      container_port = var.task_container_port
+      registry_arn = var.service_registry_arn
+      // container_port = var.task_container_port
```

Comment:
Please remove this change and merge master into your PR, this is fixed by a conditional.